src/chainstate/stacks/boot/costs.clar

```diff
@@ -232,6 +232,21 @@
 (define-read-only (cost_xor (n uint))
     (runtime u1000))
 
+(define-read-only (cost_bitwise_and (n uint))
```

Comment:
Please remove all changes to `costs.clar`.  It's consensus-critical data that cannot be changed.

Reply:
Done